### ISSUE TITLE
Adding ngxtensions to new angular v17 app fails on npm install

### ISSUE THREAD

**Issue by Pascalmh** [nartc/ngxtension-platform#143]
### How to reproduce

```bash
$ ng new example-angular-app
$ cd example-angular-app
$ npm i ngxtension
```

### The Error

```bash
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: example-angular-app@0.0.0
npm ERR! Found: @angular/common@17.0.2
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"^17.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"^16.0.0" from ngxtension@1.0.0
npm ERR! node_modules/ngxtension
npm ERR!   ngxtension@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

### Additional Info

```
$ npm list
example-angular-app@0.0.0
├── @angular-devkit/build-angular@17.0.0
├── @angular/animations@17.0.2
├── @angular/cli@17.0.0
├── @angular/common@17.0.2
├── @angular/compiler-cli@17.0.2
├── @angular/compiler@17.0.2
├── @angular/core@17.0.2
├── @angular/forms@17.0.2
├── @angular/platform-browser-dynamic@17.0.2
├── @angular/platform-browser@17.0.2
├── @angular/platform-server@17.0.2
├── @angular/router@17.0.2
├── @angular/ssr@17.0.0
├── @types/express@4.17.21
├── @types/jasmine@5.1.2
├── @types/node@18.18.9
├── express@4.18.2
├── jasmine-core@5.1.1
├── karma-chrome-launcher@3.2.0
├── karma-coverage@2.2.1
├── karma-jasmine-html-reporter@2.1.0
├── karma-jasmine@5.1.0
├── karma@6.4.2
├── rxjs@7.8.1
├── tslib@2.6.2
├── typescript@5.2.2
└── zone.js@0.14.2
```

**Comment by nartc** [comment]
ah snap I forgot to update the range of peer deps :( will release a fix asap. thanks for reporting

**Comment by nartc** [comment]
Released 1.0.3 (apparently I don't know how to specify range peer deps 🫠)
